.circleci/config.yml

```diff
@@ -205,12 +225,65 @@ jobs:
             errors=$(jq '.errors' planet4-docker-compose/artifacts/pa11y/pa11y-ci-results.json)
             if [ "$errors" -gt 0 ]; then echo "$errors errors found, see report in artifacts."; else echo "No errors, report available in artifacts."; fi
 
-  create-release-zip:
+  request-instance:
+    environment:
+      GOOGLE_PROJECT_ID: planet-4-151612
     docker:
-      - image: mhart/alpine-node
+      - image: greenpeaceinternational/p4-builder:latest
         auth: &docker_auth
     steps:
-      - run: apk add curl zip git openssh-client jq python2 make g++
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: "Check if this is for an open PR."
+          command: |
+            if [ -z "$CIRCLE_PULL_REQUEST" ];then
+              echo echo "No PR, skipping instance deploy"
+              exit 1
```

Comment:
Still need to determine what we do for this job when there is no open PR. Probably best to make it fail?

Reply:
can we do a `step halt` or something that would stop this process without failing the whole workflow ?

Reply:
Wouldn't `step halt` make the job pass and start the next jobs too?

Reply:
Absolutely no idea, didn't check the docs yet :D

Reply:
But the pipeline cannot really pass without these tests, so I guess failure is the right status? If you push a branch without a PR it will fail on that job, but you can create a PR and re-run from fail and it should work.

Reply:
Found more info on `halt` on the forum than in the docs btw https://discuss.circleci.com/t/anyway-to-terminate-circleci-entirely/27486

Reply:
We could try allowing `request-instance` to fail :thinking:

Reply:
Though it would probably still show the workflow as failed in GitHub